.travis.yml

```diff
@@ -6,9 +6,14 @@ node_js:
 #   chrome: stable
 # before_install:
 # - google-chrome-stable --headless --disable-gpu --remote-debugging-port=9222 http://localhost &
-before_install: .travis/before_install.sh
-after_success: .travis/after_success.sh
-# script: npm test && npm run lint && cd packages/core && npm run demo:build
+before_install:
+  - chmod +x .travis/before_install.sh
```

Comment:
You should only have to do this once on your computer before pushing the file. Flags should copy. 🤞